src/cls/rbd/cls_rbd_types.h

```diff
@@ -364,6 +364,34 @@ struct TrashImageSpec {
 };
 WRITE_CLASS_ENCODER(TrashImageSpec);
 
+enum MirrorImageMapState {
```

Comment:
Let's drop this for now -- right now, the existence of the struct indicates the image is associated (or dissociating or associating).